.github/workflows/publish-template.yml

```diff
@@ -0,0 +1,83 @@
+# This is a template used by the publish-packages workflow
+
+name: Publish template workflow
+
+on:
+  workflow_call:
+    inputs:
+      path:
+        required: true
+        type: string
+      lib-name:
+        required: true
+        type: string
+      cdn:
+        required: false
+        type: boolean
+
+    secrets:
+      NPM_TOKEN:
+        required: true
+      CDN_EUNO_SAS:
+        required: false
+      CDN_USWE_SAS:
+        required: false
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.path }}/${{ inputs.lib-name }}
+
+    steps:
+        - uses: actions/checkout@v2
+
+        - uses: actions/setup-node@v1
+
+        - name: Install dependencies
+          run: npm install
+
+        # Runs lerna bootstrap scoped to this package and any local dependencies so that npm run build:all commands work
+        - name: Link dependencies
+          run: npm run link:localDeps
```

Comment:
FYI you can pass `--if-present` to this command to not error out if a given package doesn't have this script defined.

Reply:
Good to know, thanks! Added.